tests/core/test_views_errors.py

```diff
@@ -40,19 +40,16 @@ def test_backoff_header_is_present_on_error_responses(self):
             self.assertEquals(response.headers['Backoff'], '10')
 
     def test_404_is_valid_formatted_error(self):
-        response = self.app.get('/unknown', status=404)
-        self.assertFormattedError(response, 404, ERRORS.MISSING_RESOURCE, "Not Found",
-                                  "The resource you are looking for could not be found.")
-
-    def test_404_can_be_overridded(self):
         custom_404 = http_error(httpexceptions.HTTPNotFound(),
                                 errno=ERRORS.MISSING_RESOURCE,
-                                message="Customized.")
+                                message="Customized.",
+                                details={"id": "abc", "resource_name": "collection"})
         with mock.patch('tests.core.testapp.views.Mushroom._extract_filters',
                         side_effect=custom_404):
             response = self.app.get(self.sample_url, headers=self.headers, status=404)
         self.assertFormattedError(
-            response, 404, ERRORS.MISSING_RESOURCE, "Not Found", "Customized.")
+            response=response, code=404, errno=ERRORS.MISSING_RESOURCE, error="Not Found",
+            message="Customized.", details={"id": "abc", "resource_name": "collection"})
```

Comment:
I don't think this test is relevant to what you want to change. This looks like a test for the underlying `kinto.core` resource code and specifically like it's trying to verify that if a resource has an `_extract_filters` method that raises an exception, that exception gets passed through correctly.

I think you need tests to verify that getting a non-existent bucket, collection, or record has the correct details attached. You might look for those tests in `tests/test_views_*.py`.